<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetClass</class>
 <widget class="QWidget" name="WidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>325</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton_open_com">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>30</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OPEN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_test_network">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TEST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear_discr">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>107</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear_sloving">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>217</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_port_num">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_baudrate">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>30</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>114</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>describe:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>224</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>slove:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_discr">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>580</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sloving">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>580</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Widget.qrc"/>
 </resources>
 <connections/>
</ui>
